install/docker/docker-compose.yaml

```diff
@@ -15,75 +20,81 @@ services:
       - $HOME/.minikube:$HOME/.minikube:ro
     ports:
       - "9081:9081"
-  meshery-istio:
-    image: layer5/meshery-istio:stable-latest
-    labels:
-      - "com.centurylinklabs.watchtower.enable=true"
-    ports:
-      - "10000:10000"
-  meshery-linkerd:
-    image: layer5/meshery-linkerd:stable-latest
-    labels:
-      - "com.centurylinklabs.watchtower.enable=true"
-    ports:
-      - "10001:10001"
-  meshery-consul:
-    image: layer5/meshery-consul:stable-latest
-    labels:
-      - "com.centurylinklabs.watchtower.enable=true"
-    ports:
-      - "10002:10002"
-  meshery-nsm:
-    image: layer5/meshery-nsm:stable-latest
-    labels:
-      - "com.centurylinklabs.watchtower.enable=true"
-    ports:
-      - "10004:10004"
-  meshery-app-mesh:
-    image: layer5/meshery-app-mesh:stable-latest
-    labels:
-      - "com.centurylinklabs.watchtower.enable=true"
-    ports:
-      - "10005:10005"
-  meshery-traefik-mesh:
-    image: layer5/meshery-traefik-mesh:stable-latest
-    labels:
-      - "com.centurylinklabs.watchtower.enable=true"
-    ports:
-      - "10006:10006"
-  meshery-kuma:
-    image: layer5/meshery-kuma:stable-latest
-    labels:
-      - "com.centurylinklabs.watchtower.enable=true"
-    ports:
-      - "10007:10007"
-  meshery-osm:
-    image: layer5/meshery-osm:stable-latest
-    labels:
-      - "com.centurylinklabs.watchtower.enable=true"
-    ports:
-      - "10009:10009"
-  meshery-nginx-sm:
-    image: layer5/meshery-nginx-sm:stable-latest
-    labels:
-      - "com.centurylinklabs.watchtower.enable=true"
-    ports:
-      - "10010:10010"
-  meshery-cilium:
-    image: layer5/meshery-cilium:stable-latest
-    labels:
-      - "com.centurylinklabs.watchtower.enable=true"
+  meshery-db:
+    image:  postgres:10.22-alpine
+    environment:
+      - "POSTGRES_PASSWORD=meshery"
```

Comment:
```suggestion
      - "DATABASE_PASSWORD=meshery"
```

Reply:
This is a postgres specific environment variable. Postgres expects it to be POSTGRES_PASSWORD